qase-python-commons/src/qase/commons/config.py

```diff
@@ -59,3 +59,6 @@ def _get_config(self, key):
         for key in keys[:-1]:
             config = config.get(key, {})
         return config.get(keys[-1], None)
+
+    def to_json(self):
+        return json.dumps(self.config, indent=4)
```

Comment:
Maybe it makes sense to add `sort_keys = True` to make the output more uniform and, thus, easier to read.

Reply:
Done